src/wavefront-obj-parser.js

```diff
@@ -2,47 +2,50 @@ module.exports = ParseWavefrontObj
 
 // Map .obj vertex info line names to our returned property names
 var vertexInfoNameMap = {v: 'vertex', vt: 'uv', vn: 'normal'}
-// The returned properties that we will populate
-var parsedProperties = ['normal', 'uv', 'vertex', 'normalIndex', 'uvIndex', 'vertexIndex']
+
 function ParseWavefrontObj (wavefrontString) {
+  'use strict'
```

Comment:
It flags the function to be executed in [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) (MDN) which allows the js engine to performs specific optimizations. For this specific code base, on node 6.x and without code warmup (i.e. parsing multiple models) the benchmark did not noted any actual improvement, but on other js engine (latest node, future node versions, browsers...) and/or with some code warmup to trigger V8 code optimization there might actually be some improvement.

Personally, I always enforce `strict mode` as there are no real downside to it.